boards/microbit_v2/Makefile

```diff
@@ -17,11 +18,19 @@ install: flash
 
 .PHONY: flash-debug
 flash-debug: $(TOCK_ROOT_DIRECTORY)target/$(TARGET)/debug/$(PLATFORM).elf
-	$(OPENOCD) $(OPENOCD_OPTIONS) -c "program $<; verify_image $<;  reset; shutdown;"
+ifdef USE_PROBE_RS
+	$(PROBE_RS) download --chip nRF52833_xxAA --verify $<	
+else
+	$(OPENOCD) $(OPENOCD_OPTIONS) -c "program $<; verify_image $<; reset; shutdown;"
+endif
 
 .PHONY: flash
 flash: $(TOCK_ROOT_DIRECTORY)target/$(TARGET)/release/$(PLATFORM).elf
+ifdef USE_PROBE_RS
+	$(PROBE_RS) download --chip nRF52833_xxAA --verify $<	
+else
 	$(OPENOCD) $(OPENOCD_OPTIONS) -c "program $<; verify_image $<; reset; shutdown;"
+endif
 
```

Comment:
These should just be different targets. `flash` and `program` are just legacy words we use, but we can have more so that each target uses a specific code loading tool. `make install` should choose the "default". Anyone who wants to use something else can just use `make install/program/probe/etc`. I don't have a preference on what the default is. But I would like to avoid `ifdef`, especially within a recipe.